ui/components/multichain/account-overview/account-overview-tabs.tsx

```diff
@@ -65,9 +65,6 @@ export const AccountOverviewTabs = ({
       if (tabName === AccountOverviewTabKey.Nfts) {
         dispatch(detectNfts(selectedChainIds));
       }
-      if (tabName === AccountOverviewTabKey.Activity) {
-        dispatch(updateIncomingTransactions());
```

Comment:
Removing this since the Activity view component already handles starts polling on mount. This one causes a duplicate